### PR TITLE
DSET-3658 Cross-team query support

### PR DESCRIPTION
**Pull request by dlobue**
First pass at adding MVP cross-team support to grafana datasource.

DSET-3658

### CODE REVIEW

File: pkg/plugin/plugin.go [hunk 1]
```diff
@@ -57,11 +57,12 @@ func (d *DataSetDatasource) QueryData(ctx context.Context, req *backend.QueryDat
...
-	QueryType           string  `json:"queryType"`
-	Format              string  `json:"format"`
-	BreakDownFacetValue *string `json:"breakDownFacetValue"`
-	Label               *string `json:"label"`
+	Expression          string    `json:"expression"`
+	QueryType           string    `json:"queryType"`
+	Format              string    `json:"format"`
+	BreakDownFacetValue *string   `json:"breakDownFacetValue"`
+	Label               *string   `json:"label"`
+	AccountEmails       *[]string `json:"accountEmails"`
```
Comment (jmakar-scalyr) [inline pkg/plugin/plugin.go:65]: Slices (ie `[]<type>`) are reference types so you can assign nil to them inherently
```
	var a []int = []int{1, 2, 3}
	var b []int = []int{}
	var c []int = nil
```

Better to have the AccountEmails type to be just `[]string`

**Review by jmakar-scalyr — CHANGES_REQUESTED** [review]
Looks good, just a minor type adjustment to fix

A brief blurb and example screenshots in the README.md would be helpful.  Also good to add a note about this in the CHANGELOG.md as well.

**Comment by dlobue** [comment]
@jmakar-scalyr all requested changes have been made!

**Review by jmakar-scalyr — APPROVED** [review]
Looks great!

**Pull request merged**